book.rb

```diff
@@ -1,16 +1,16 @@
 require_relative 'rental'
 
 class Book
-  def initialize(title, author)
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] to make the associations work Getters and Setters for @rentals should be present